core/modules/path/path.path.inc

```diff
@@ -14,8 +14,8 @@
 function node_path_info() {
   $info['node'] = array(
     'entity type' => 'node',
-    'label' => t('Content paths'),
-    'pattern description' => t('Default path pattern (applies to all content types with blank patterns below)'),
+    'label' => t('Content'),
```

Comment:
"Content" is describing "paths" here. If we remove "paths" it should be replaced with another word of some sort. "URLs" or "patterns".